### PR TITLE
spotify-unwrapped: 1.1.46.916.g416cacf1 -> 1.1.55.494.gca75f788

### PR DESCRIPTION
**Pull request by samuela**
###### Motivation for this change
Upgrades spotify-unwrapped from 1.1.46.916.g416cacf1 to 1.1.55.494.gca75f788

This PR was automatically generated by [nixpkgs-upkeep](https://github.com/samuela/nixpkgs-upkeep).
- [CI workflow](https://github.com/samuela/nixpkgs-upkeep/actions/runs/666321468) that created this PR.
- Internal tag: 96fe581eb7f2ba2387b11e0da5dec28e.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 116805` at de7418b6 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built successfully:</summary>
  <ul>
    <li>spotify</li>
    <li><a href="https://gist.github.com/ea85d57ba1f2fb6e9169b9acd703ae41">spotify-unwrapped</a></li></li>
    <li><a href="https://gist.github.com/b0955baf6f1de04ca621954b5651ddc6">spotifywm</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/build-tools-in-build-inputs.md">build-tools-in-build-inputs</a>

squashfsTools is a build tool so it likely goes to `nativeBuildInputs`, not `buildInputs`.

Near pkgs/applications/audio/spotify/default.nix:85:3:
```
   |
85 |   buildInputs = [ squashfsTools ];
   |   ^
```
</li>
  </ul>
</details>

**Comment by samuela** [comment]
Thanks @r-rmcgibbo for the suggestion! I fixed it in 3b84765.

The new version wasn't running for me so I fixed (what I believe to be) all the dependency issues in 78d3a4b.

File: pkgs/applications/audio/spotify/default.nix
Comment (SuperSandro2000) [inline pkgs/applications/audio/spotify/default.nix:87]: Does stripping break it?

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 116805` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
3 packages built:
</summary>
<ul>
<li>spotify</li>
<li>spotify-unwrapped</li>
<li>spotifywm</li>
</ul>
</details>

File: pkgs/applications/audio/spotify/default.nix
Comment (samuela) [reply pkgs/applications/audio/spotify/default.nix:87]: I'm not sure. This was just the suggestion in @r-rmcgibbo's comment.

Comment (samuela) [reply pkgs/applications/audio/spotify/default.nix:87]: Oops, I just realized you commented on line 87, not 84-85... That's a great question re `dontStrip`. That decision was before my time... Perhaps one of the maintainers would know?

**Review by ajs124 — APPROVED** [review]

File: pkgs/applications/audio/spotify/default.nix
Comment (timokau) [reply pkgs/applications/audio/spotify/default.nix:87]: Good catch. This was already included in the original commit (f81bc24790895bb06e3ead3d684042e94fc2c87a). I'd say there is a good chance that it is not needed (anymore). I think we should generally not require unrelated improvements on update PRs though.

**Review by timokau — APPROVED** [review]
Looks good to me, thanks! I won't merge this yet because I don't want to step over @SuperSandro2000 regarding the `dontStrip` thread. In my opinion it is good to go though.

File: pkgs/applications/audio/spotify/default.nix
Comment (samuela) [reply pkgs/applications/audio/spotify/default.nix:87]: Yeah, I'd be happy to submit a follow-up PR removing `dontStrip`

Comment (SuperSandro2000) [reply pkgs/applications/audio/spotify/default.nix:87]: So we don't need it?

```suggestion
```

**Pull request merged**
